### PR TITLE
2x3 card grid, font resizing

### PR DESCRIPTION
**Pull request by jordan2lee**
Addresses Issue #24 

- 2x3 widget card grid
- font sizing and style changes
- figure color changes
- refactor card code for better readability 

New look
image

### CODE REVIEW

File: bmeg_app/views/home_view.py [hunk 1]
```diff
@@ -25,68 +25,89 @@
 # Page
 #######
 NAME=i18n.t('app.config.tabname_widget_home')
+
+cluster_card = dbc.Card(
```
Comment (kellrott) [inline bmeg_app/views/home_view.py:29]: Is there a way to get this set of cards built by responding to a config file?

Comment (jordan2lee) [reply bmeg_app/views/home_view.py:29]: my first thought would be to create a function that inputs header and body string, then returns the full `dbc.Card()`. I had previously started prototyping this function and could include it here

**Comment by bwalsh** [comment]
I have code in our gen3 branch that creates 'cards' from yml file

**Comment by bwalsh** [comment]
https://github.com/ohsu-comp-bio/hatchery-lite/blob/master/workspace_cards.yaml
https://github.com/ohsu-comp-bio/hatchery-lite/blob/master/app.py#L12-L19

**Pull request merged**